moduleroot/spec/spec_helper.rb.erb

```diff
@@ -43,6 +43,9 @@ RSpec.configure do |c|
   <%- if @configs['default_facter_version'] -%>
   c.default_facter_version = '<%= @configs['default_facter_version'] %>'
   <%- end -%>
+  <%- if @configs['facter_implementation'] -%>
+  c.facter_implementation = <%= @configs['facter_implementation'] %>
```

Comment:
Should this have single quotes like the other options?

Reply:
No because the inlut is something like `:rspec` so quotes won't work there